packages/ui/src/components/shadcn/ui/badge.tsx

```diff
@@ -28,12 +28,16 @@ export interface BadgeProps
   extends React.HTMLAttributes<HTMLDivElement>,
     VariantProps<typeof badgeVariants> {}
 
-function Badge({ className, variant = 'default', children, ...props }: BadgeProps) {
-  return (
-    <div className={cn(badgeVariants({ variant }), className)} {...props}>
-      {children}
-    </div>
-  )
-}
+// Forward refs in order to allow tooltips to be applied to the badge
```

Comment:
As it says in the comment. In other words:

I couldn’t use Badge `aschild` in a Tooltip without this. The alternative is to wrap `Badge` in a `span` but that felt messy.